src/test/groovy/ratpack/zipkin/ServerTracingModuleSpec.groovy

```diff
@@ -368,4 +371,49 @@ class ServerTracingModuleSpec extends Specification {
 		then:
 			assertThat(reporter.getSpans()).isNotEmpty()
 	}
+
+	def 'Should nest client span and server span should have the same trace id'() {
+		given:
+			def webServer = new MockWebServer()
+			webServer.start()
+			webServer.enqueue(new MockResponse().setResponseCode(200))
+			def url = webServer.url("/")
```

Comment:
@llinder I do want to eventually use the same approach in the http client tests, as you suggested. Didn't do it in this PR because I haven't figured out a nice way to get Guice to give me an instance of `ZipkinHttpClientImpl`. Here, because it's in an embedded app, it's pretty straightforward.